pkg/cache/snapshot.go

```diff
@@ -157,26 +159,41 @@ func (c *Cache) Snapshot(ctx context.Context) (*Snapshot, error) {
 
 // snapshotClusterQueue creates a copy of ClusterQueue that includes
 // references to immutable objects and deep copies of changing ones.
-func snapshotClusterQueue(c *clusterQueue) *ClusterQueueSnapshot {
+func (c *Cache) snapshotClusterQueue(ctx context.Context, cq *clusterQueue) *ClusterQueueSnapshot {
```

Comment:
If you rename the c as cq, please submit a preparatory PR to make the diff easier to dijest.

Reply:
I propose not, because it blows the size of the diff unnecesarily

Reply:
IIUC this is already addressed, please rebase and feel free to resolve

Reply:
Done